### PR TITLE
Fixes and updates for Ubuntu Touch port

### PR DESCRIPTION
**Pull request by mrkubax10**
Updated to UT 20.04 and Clickable 7

### CODE REVIEW

File: .github/workflows/ubuntu-touch.yml [hunk 1]
```diff
@@ -92,12 +92,12 @@ jobs:
...
-                                         --arch ${ARCH}
+      #- name: Publish to Open Store
+      #  if: ${{ github.ref == 'refs/heads/master' && matrix.build_type == 'Release' && matrix.opengl == 'sdl' }}
+      #  env:
+      #    ARCH: ${{ matrix.arch }}
+      #    OPENSTORE_KEY: ${{ secrets.OPENSTORE_KEY }}
+      #  run: |
+      #    ~/.local/bin/clickable publish "* $(git log -1 --pretty=%B |         \
+      #                                   head -1)" --apikey ${OPENSTORE_KEY}   \
+      #                                   --arch ${ARCH}
```
Comment (HybridDog) [inline .github/workflows/ubuntu-touch.yml:103]: Is the disabling of this `Publish to Open Store` task intentional?

Comment (mrkubax10) [reply .github/workflows/ubuntu-touch.yml:103]: Yes, because I think that only releases should be published. You can see at what state game currently is.

Comment (HybridDog) [reply .github/workflows/ubuntu-touch.yml:103]: I agree.

**Review by HybridDog — APPROVED** [review]
The changes look fine for me. 
I know almost nothing about Ubuntu Touch, so my approval may not count for much.

**Comment by mrkubax10** [comment]
This PR uses SDL2 from Ubuntu focal repositories which doesn't support Mir anymore and some devices still use it. To make UT builds at least compiling it should be merged. I will add Mir support to latest SDL2 and then create supplementary PR which will make use of that patched SDL2.

**Comment by zubozrout** [comment]
Is the project currently buildable with clickable?
Simply running `clickable build --arch arm64` it ends up with the following for me when I try:
```
[100%] Built target tinygettext
Install the project...
-- Install configuration: "Release"
-- Up-to-date: supertux/build.clickable/tinygettext/lib/libtinygettext.a
CMake Error at cmake_install.cmake:58 (file):
  file cannot create directory: /tinygettext.  Maybe need administrative
  privileges.

make[3]: *** [Makefile:100: install] Error 1
make[2]: *** [CMakeFiles/tinygettext_project.dir/build.make:102: tinygettext_project-prefix/src/tinygettext_project-stamp/tinygettext_project-install] Error 2
make[1]: *** [CMakeFiles/Makefile2:169: CMakeFiles/tinygettext_project.dir/all] Error 2
make: *** [Makefile:156: all] Error 2
```

**Comment by mrkubax10** [comment]
CI is building without issues so it should work.

**Comment by zubozrout** [comment]
I didn't notice UT is also in the CI, thank you so much :). Still weird it fails building for me though - I can't see any difference quickly. But atm this is enough as my only goal was to install a Focal build on UT since it is no longer being updated in the OpenStore.

**Comment by mrkubax10** [comment]
> But atm this is enough as my only goal was to install a Focal build on UT since it is no longer being updated in the OpenStore.

It will be uploaded to OpenStore when 0.7.0 is released.

**Pull request merged**